package.json

```diff
@@ -34,12 +34,13 @@
     "gulp-mocha": "4.3.1",
     "mocha": "3.2.0",
     "mocha-typescript": "1.0.22",
-    "typescript": "2.3.3"
+    "typescript": "2.3.3",
+    "path": "0.12.7"
```

Comment:
Just trying to appease the darned CI, will clean up once I figure out the issue.